### PR TITLE
Change to colorsys

### PR DESCRIPTION
**Pull request by FoamyGuy**
I believe this are the necessary changes for this to get moved over to being colorsys.

I updated the gitignore, removed the cpython file, added the example script.

I don't believe I have access to rename the repo, but I think that is ready to be done after this is merged.

### CODE REVIEW

**Comment by FoamyGuy** [comment]
It seems that we have some trouble building the docs. 
```
Warning, treated as error:
/home/runner/work/Adafruit_CircuitPython_CPython/Adafruit_CircuitPython_CPython/docs/index.rst:17:toctree contains reference to document 'api' that doesn't have a title: no link will be generated
Error: Process completed with exit code 2.
```

I'm not really sure what this error is trying to indicate. I thought this SO question might be relavent: https://stackoverflow.com/questions/14079655/cannot-get-toctree-in-sphinx-to-show-link but I'm not sure after poking around some. Our other libraries don't seem to have titles in the rst files like the answer here is suggesting and they still build okay. 

I will take a closer look in the next few days and try to make this match another libraries docs setup.

File: adafruit_cpython.py [hunk 1]
```diff
@@ -1,26 +0,0 @@
-# SPDX-FileCopyrightText: 2017 Scott Shawcroft for Adafruit Industries
-#
-# SPDX-License-Identifier: MIT
-
-"""
-`adafruit_cpython`
-====================================================
```
Comment (tannewt) [inline adafruit_cpython.py:7]: This is the title that was used before. I think you'll want something like this in colorsys.py.

**Comment by FoamyGuy** [comment]
Thank you! got it building now.

**Comment by jposada202020** [comment]
So happy for this :)

**Review by tannewt — APPROVED** [review]
Thanks!

**Comment by evaherrada** [comment]
@tannewt Should I migrate the library name and everything to colorsys? I assume I need to change it in settings, then update the readthedocs thing and make sure it's deploying to pypi correctly

**Comment by tannewt** [comment]
@dherrada Yes please!

**Pull request merged**